Source/lua/randomat2/events/gravity.lua

```diff
@@ -0,0 +1,77 @@
+local EVENT = {}
+
+CreateConVar("randomat_gravity_timer", 30, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "Timer (def. 30)", 5,90)
+CreateConVar("randomat_gravity_duration", 3, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "Duration (def. 3)", 1,15)
+CreateConVar("randomat_gravity_minimum", 70, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "The minimum gravity (def. 70)",10,500)
+CreateConVar("randomat_gravity_maximum", 2000, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "The minimum gravity (def. 70)",700,4000)
+
+EVENT.Title = "I don't think you realise the gravity of the situation."
+EVENT.id = "gravity"
+EVENT.AltTitle = "Gravity Changer"
+
+local defaultGravity = 600
+
+local function SetGravity(increase)
+    if increase then
+        RunConsoleCommand("sv_gravity", GetConVar("randomat_gravity_maximum"):GetFloat())
+    else
+        RunConsoleCommand("sv_gravity", GetConVar("randomat_gravity_minimum"):GetFloat())
+    end
+    return not increase
+end
+
+function EVENT:Begin()
+    defaultGravity = GetConVar("sv_gravity"):GetFloat()
+    if SERVER then
+        SetGravity(true)
+        timer.Simple(duration, function()
```

Comment:
`duration` isn't defined here

Reply:
Good catch, I didn't originally have that as a timer and forgot I used the variable there